Library/Homebrew/cmd/test-bot.rb

```diff
@@ -549,15 +554,13 @@ def test(*args)
     end
 
     def check_results
-      status = :passed
-      steps.each do |step|
+      steps.all? do |step|
```

Comment:
@mikemcquaid I did a little refactoring here. Do you want I submit another PR just for this code refactoring.